content/docs/command-reference/add.md

```diff
@@ -368,9 +363,10 @@ data.xml data.xml.dvc
 ```
 
 The resulting `.dvc` file will save the provided local `path` as if the data was
-already in the workspace, while the `md5` hash points to the copy of the data
-that has now been transferred to the <abbr>cache</abbr>. Let's check the
-contents of `data.xml.dvc` in this case:
```

Comment:
We can clarify that if links are set properly `data.xml` is "virtual" and doesn't take any space

Reply:
Good call. Mentioned in the 2nd paragraph and here.